### PR TITLE
Add IMU details and diagrams

### PR DESCRIPTION
**Pull request by yellowjaguar5**
These changes were split from #200.

### CODE REVIEW

File: source/docs/hardware/sensors/accelerometers-hardware.rst [hunk 1]
```diff
@@ -5,31 +5,42 @@ Accelerometers are common sensors used to measure acceleration.
...
-
-  - `Analog Devices ADIS16448 and ADIS 16470 IMUs <https://www.analog.com/en/landing-pages/001/first.html>`__
-  - `CTRE Pigeon IMU <https://www.ctr-electronics.com/gadgeteer-imu-module-pigeon.html>`__
-  - `Kauai Labs NavX <https://pdocs.kauailabs.com/navx-mxp/>`__
 
 Types of accelerometers
 -----------------------
 
-There are two types of accelerometers commonly-used in FRC: single-axis accelerometers, and multi-axis accelerometers.
+There are three types of accelerometers commonly-used in FRC: single-axis accelerometers, multi-axis accelerometers and IMUs.
```
Comment (AustinShalit) [inline source/docs/hardware/sensors/accelerometers-hardware.rst:14]:
```suggestion
There are three types of accelerometers commonly-used in FRC: single-axis accelerometers, multi-axis accelerometers, and IMUs.
```

File: source/docs/hardware/sensors/gyros-hardware.rst [hunk 2]
```diff
@@ -14,22 +14,24 @@ Several popular FRC devices (known as "inertial measurement units," or "IMUs") c
...
 Three-axis gyros
 ^^^^^^^^^^^^^^^^
 
-.. todo:: add picture
+.. image:: images/gyros-hardware/triple-axis-i2c-gyro-to-roborio.svg
 
-Three-axis gyros measure rotation rate around all three spacial axes (typically labeled x, y, and z).
+.. image:: images/gyros-hardware/yaw-pitch-roll.png
+
+Three-axis gyros measure rotation rate around all three spacial axes (typically labeled x, y, and z). The motion around these axis is called pitch, yaw and roll.
```
Comment (AustinShalit) [inline source/docs/hardware/sensors/gyros-hardware.rst:35]:
```suggestion
Three-axis gyros measure rotation rate around all three spacial axes (typically labeled x, y, and z). The motion around these axis is called pitch, yaw, and roll.
```

File: source/docs/hardware/sensors/gyros-hardware.rst [hunk 3]
```diff
@@ -14,22 +14,24 @@ Several popular FRC devices (known as "inertial measurement units," or "IMUs") c
 Types of gyros
```
Comment (Daltz333) [inline source/docs/hardware/sensors/gyros-hardware.rst:14]:
```suggestion
Types of Gyros
```

File: source/docs/hardware/sensors/gyros-hardware.rst [hunk 4]
```diff
@@ -14,22 +14,24 @@ Several popular FRC devices (known as "inertial measurement units," or "IMUs") c
...
 --------------
 
-There are two types of Gyros commonly-used in FRC: single-axis gyros, and three-axis gyros.
+There are two types of Gyros commonly-used in FRC: single-axis gyros, three-axis gyros and IMUs, which often include a 3-axis gyro.
 
 Single-axis gyros
```
Comment (Daltz333) [inline source/docs/hardware/sensors/gyros-hardware.rst:19]:
```suggestion
Single-axis Gyros
```

File: source/docs/hardware/sensors/gyros-hardware.rst [hunk 5]
```diff
@@ -14,22 +14,24 @@ Several popular FRC devices (known as "inertial measurement units," or "IMUs") c
...
+As per their name, single-axis gyros measure rotation rate around a single axis.  This axis is generally specified on the physical device, and mounting the device in the proper orientation so that the desired axis is measured is highly important.  Some single-axis gyros can output an analog voltage corresponding to the measured rate of rotation, and so connect to the roboRIO's :doc:`analog input <analog-inputs-hardware>` ports. Other single-axis gyros, such as as the `ADXRS450 <https://wiki.analog.com/first/first_robotics_donation_resources>`__ pictured above, use the :ref:`SPI port <docs/hardware/sensors/serial-buses:SPI>` on the roboRIO instead.
 
 The `Analog Devices ADXRS450 FRC Gyro Board <https://www.analog.com/en/landing-pages/001/first.html>`__ that has been in FIRST Choice in recent years is a commonly used single axis gyro.
 
 Three-axis gyros
```
Comment (Daltz333) [inline source/docs/hardware/sensors/gyros-hardware.rst:28]:
```suggestion
Three-axis Gyros
```

File: source/docs/hardware/sensors/gyros-hardware.rst [hunk 6]
```diff
@@ -14,22 +14,24 @@ Several popular FRC devices (known as "inertial measurement units," or "IMUs") c
...
 ^^^^^^^^^^^^^^^^
 
-.. todo:: add picture
+.. image:: images/gyros-hardware/triple-axis-i2c-gyro-to-roborio.svg
 
-Three-axis gyros measure rotation rate around all three spacial axes (typically labeled x, y, and z).
+.. image:: images/gyros-hardware/yaw-pitch-roll.png
```
Comment (Daltz333) [inline source/docs/hardware/sensors/gyros-hardware.rst:33]: > .. image:: images/gyros-hardware/yaw-pitch-roll.png

Move this below the pitch, yaw, and roll line for better flow.

Comment (yellowjaguar5) [reply source/docs/hardware/sensors/gyros-hardware.rst]: Fixed in aa03d65.

File: source/docs/hardware/sensors/accelerometers-hardware.rst [hunk 7]
```diff
@@ -5,31 +5,42 @@ Accelerometers are common sensors used to measure acceleration.
...
-  - `Kauai Labs NavX <https://pdocs.kauailabs.com/navx-mxp/>`__
 
 Types of accelerometers
 -----------------------
 
-There are two types of accelerometers commonly-used in FRC: single-axis accelerometers, and multi-axis accelerometers.
+There are three types of accelerometers commonly-used in FRC: single-axis accelerometers, multi-axis accelerometers, and IMUs.
 
 Single-axis accelerometers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (Daltz333) [inline source/docs/hardware/sensors/accelerometers-hardware.rst:18]: Is there no picture of a single-axis accelerometer?

Comment (yellowjaguar5) [reply source/docs/hardware/sensors/accelerometers-hardware.rst:18]: Added in 66f2eae.

**Comment by Modelmat** [comment]
I feel like the images are too big, filling up 100% of the screen width means you can only really see one image and a sentence at a time. 

Also the yaw, pitch, and roll image seems to exist at https://www.researchgate.net/figure/Explanation-of-roll-pitch-and-yaw-movements-of-a-gyroscope_fig10_319619198 - I don't think we'd have a license to use it.

The background of it is also a slightly different white - 
image - the image is FFFFFF but the background if #FCFCFC

**Comment by yellowjaguar5** [comment]
@Modelmat Whoops, that's my bad on the yaw, pitch, and roll image. I'll make a new one. In terms of the background, I'm curious if we can render all the SVGs with a transparent background. I'll look into it.
I'll also reduce the image size.

**Comment by Modelmat** [comment]
Oh, the background was specifically for that image. I didn't see a difference for the other diagrams included.

**Comment by yellowjaguar5** [comment]
@Modelmat I believe I addressed all your comments in 9c518b9

**Pull request merged**